apps/api/src/service/authService.ts

```diff
@@ -76,3 +82,46 @@ export async function isAssociatedRep(session: Partial<SessionData>, application
 
 	return false;
 }
+
+/**
+ * Handler to enforce Request is Authorized
+ * Insures Type safety for downstream Request Handler due to limitations with Express Type definitions
+ * @param request Express request
+ * @returns boolean
+ */
+export const isAuthenticatedRequest = (request: Request): request is AuthorizedRequest => {
+	return isRequestWithSession(request);
+};
```

Comment:
We don't need this. Duplicate function with new name.

Reply:
Sure, crossed my mind too. Thought there may be value in case we wanted to expand the request handler version.